CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
+# 4.0.0 (2020-05-22)
+
+### New features
+
```

Comment:
Should we have an "Added support for @angular/core@^9." entry here? We have it in most of the other changelogs I've been looking at today, so just checking.

Reply:
Done.

Reply:
If we added that here, seems like we should have it below for the RC too. I'll add it.